src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -4,6 +4,7 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faReply } from "@fortawesome/free-solid-svg-icons";
 
 import { MessageToDisplay } from "types/chat";
+import { RenderMarkdown } from "components/organisms/RenderMarkdown";
```

Comment:
This should be sorted down with the other `components/*` imports, above the atoms.

Reply:
done

Reply:
Confirm 
![image](https://user-images.githubusercontent.com/63194656/120822425-13390100-c55f-11eb-9f78-f89ebf3b9fe9.png)

Reply:
pushed eb80b9bfcef0ee4f5e1c8abbd7d183a52087f4b1 to resolve this.
I wonder -- since imports ordering seems to be valuable, is there something like https://github.com/PyCQA/isort but for JS/TS? then it could be bolted on pre-commit/black/prettify/... to autoharmonize all the imports .  We AFAIK use that `isort` for our dandi-cli as part of the black'ing in pre-commit and it is working out nicely ;-)